src/minisatip.c

```diff
@@ -1798,20 +1828,23 @@ int main(int argc, char *argv[])
 		if ((ssdp1 = udp_bind(opts.disc_host, 1900, 1)) < 1)
 			FAIL("SSDP: Could not bind on %s udp port 1900", opts.disc_host);
 
-		si = sockets_add(ssdp, NULL, -1, TYPE_UDP, (socket_action)ssdp_reply,
+		fill_sockaddr(&sv, "0.0.0.0", 1900, 1);
+		si = sockets_add(ssdp, &sv, -1, TYPE_UDP, (socket_action)ssdp_reply,
 						 (socket_action)ssdp_byebye, (socket_action)ssdp_discovery);
-		si1 = sockets_add(ssdp1, NULL, -1, TYPE_UDP, (socket_action)ssdp_reply,
+		si1 = sockets_add(ssdp1, &sv, -1, TYPE_UDP, (socket_action)ssdp_reply,
 						  (socket_action)ssdp_byebye, (socket_action)ssdp_discovery);
 		if (si < 0 || si1 < 0)
 			FAIL("sockets_add failed for ssdp");
 	}
 
 	sockets_timeout(si, 60 * 1000);
 	set_sockets_rtime(si, -60 * 1000);
-	if (0 > sockets_add(rtsp, NULL, -1, TYPE_SERVER, (socket_action)new_rtsp,
+	fill_sockaddr(&sv, "0.0.0.0", opts.rtsp_port, 1);
```

Comment:
This is not logging related and breaks compatibility with IPv6 lets keep it separate from this PR

Reply:
Hi,

This is necessary because in the current implementation not every time the full info "address"+"socket" is printed for _any_ socket connection. Then to make the code more readable and standard I use this simple approach: all listening sockets have the address too. And the implementation works, almost for IPv4. Any idea for IPv4/6 support?

Reply:
Can you point out where the printing happens is not right (eg NULL is printed) ?
You can use NULL instead of "0.0.0.0"